libs/live-config/src/LiveConfig.ts

```diff
@@ -1,4 +1,5 @@
 import { Provider } from "./providers";
+import _ from "lodash";
```

Comment:
please do not import the whole lodash.
i think we never did it in the whole project (if we do it's a mistake) and then instead we always used the functional import.

for instance

```
import merge from "lodash/merge"
```

Reply:
https://ledgerhq.atlassian.net/browse/LIVE-11142
(unfortunately it seems we have lost track of this good practice over the last months, i created another task as a side topic on top of the fact it would be great if you could also solve this one)